lib/init.coffee

```diff
@@ -11,9 +11,21 @@ module.exports =
     extensions:
       type: 'string'
       default: 'c++,cc,cpp,cu,cuh,h,hpp'
-    cpplintExecutablePath:
+    executablePath:
       type: 'string'
-      default: path.join __dirname, '..', 'bin'
+      default: path.join __dirname, '..', 'bin', 'cpplint.py'
 
   activate: ->
+    require('atom-package-deps').install 'linter-cpplint'
     console.log 'activate linter-cpplint'
```

Comment:
ok, removed in [commit bef9b7b](https://github.com/imironhead/linter-cpplint/commit/bef9b7b461aaeae4f32b3f2306c295c650409e10).